### ISSUE TITLE
Looks like unit tests are not working using travis

### ISSUE THREAD

**Issue by slavielle** [drewm/mailchimp-api#219]
Hello, I noticed something wrong about your unit tests playing on Travis :

If you click on the "passed" badge you can see on travis (https://travis-ci.org/drewm/mailchimp-api) that tests fails on PHP 5.4, 5.5 and 5.6 (because of a missing .env file) and are passing on PHP 7, 7.1 and 7.2.

But if you go on the job page where test passes (example https://travis-ci.org/drewm/mailchimp-api/jobs/308344530) you can see "No tests executed!".

Your test badge is hence displaying a false "passing".

**Comment by drewm** [comment]
They're set in the Travis environment directly. From the build output just now:

```
Setting environment variables from repository settings
$ export MC_API_KEY=[secure]
$ export MC_LIST_ID=[secure]
```

**Comment by slavielle** [comment]
Yep, see that, its working now for PHP 5.4, 5.5 and 5.6. You know why tests are not playing on PHP 7+ versions ?
